index.js

```diff
@@ -239,12 +248,15 @@ const fixWebpackChunksIssue = ({ page, basePath, asyncJs }) => {
       });
     },
     basePath,
+    bundleName,
     asyncJs
   );
 };
 
 const saveAsHtml = async ({ page, filePath, options, route }) => {
-  const content = await page.content();
+  let content = await page.content();
+  content = content.replace(/data\-onload/g, 'onload');
```

Comment:
common name. Maybe use something more specific, like `react-snap-onload`

Reply:
Yes I can add this